Web/View/LandingPages/Show.hs

```diff
@@ -84,4 +92,24 @@ renderParagraphQuote paragraphQuote =
         , subtitle = paragraphQuote.subtitle
         , imageUrl = paragraphQuote.imageUrl |> fromMaybe ""
         }
-        |> Web.Element.Quote.render
\ No newline at end of file
+        |> Web.Element.Quote.render
+
+renderParagraphHeroImage :: ParagraphHeroImage -> Html
+renderParagraphHeroImage paragraphHeroImage =
+    let
+        button = case paragraphHeroImage.link of
+            Just linkText ->
+                Just RenderButton
+                    { text = "Read More"
+                    , url = linkText
+                    , isPrimary = True
+                    }
+            _ -> Nothing
+    in
+    RenderHeroImage
+        { title = paragraphHeroImage.title
+        , subtitle = paragraphHeroImage.subtitle |> fromMaybe ""
```

Comment:
if subtitle is maybe, then we should call it `maybeSubtitle`, and value could be `Maybe Text`

In hsx if you print `['hsx|{Nothing}|]` it's the same like printing `""` -- it won't print anything

Reply:
`renderedButton = maybeButton |> fmap Web.Element.Button.render |> fromMaybe ""`
```
subTitleWrapped = maybeSubtitle 
            |> fmap (\subtitle -> cs subtitle 
                |> wrapTextFontWeight FontWeightMedium 
                |> wrapTextResponsiveFontSize TextSizeXl
            ) 
            |> fromMaybe ""
```
Did like this, it works, but it still renders button without url and empty wrappers for subtitle, am I missing something?

Reply:
https://github.com/amitaibu/ihp-cms-starter/pull/51#pullrequestreview-2188571696